### PR TITLE
Testimonials

### PR DESCRIPTION
**Pull request by mikkelklokkerud**
- Add frontend and sanity schema (prod deployed) for testimonials section
- Minor clean-ups here and there

Note that since the page category and service pages are not SSR, using string interpolation in static page queries doesn't work (graqhql fragments work but it would result in code duplication)

### CODE REVIEW

**Comment by klyngen** [comment]
💻 https://testimonia-852.review.test-alv.no

File: packages/website/src/components/testimonialsSlider/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { GatsbyImage } from 'gatsby-plugin-image';
+import Slider from 'react-slick';
+import config from '../../config';
+import { BlockContent, Title } from 'shared-components';
+
+const TestiminialSlider = ({ testimonials, heading }) => {
```
Comment (klyngen) [inline packages/website/src/components/testimonialsSlider/index.js:7]: Testiminial eller Testimonial? :+1:  ei litta trykkleif?

File: packages/website/src/templates/category.js [hunk 2]
```diff
@@ -106,6 +96,20 @@ export default Category;
 export const query = graphql`
   query($slug: String!) {
     sanityCategoryPage(slug: { current: { eq: $slug } }) {
+      testimonialSlider {
```
Comment (klyngen) [inline packages/website/src/templates/category.js:99]: Was it a good approach to pull these into separate variables for easy reuse? You had some experience, but cannot completely remember the conclusion. Just resolve the comment if it does not make sense

**Comment by klyngen** [comment]
En mere generel kommentar. Det er sikkert dataset som gjør det, men  ta en titt på denne siden: https://testimonia-852.review.test-alv.no/om-oss/livet-i-alv. Tror kanskje det er noe SSR som tryner.

**Comment by klyngen** [comment]
Ei litta klype pirk fra min side ellers bra greier kodemessig. Har du laget en egen tag for datasettet? Klarte ikke helt å finne testimonials noe steder på siden for å teste funksjonelt.

**Comment by klyngen** [comment]
💻 https://testimonia-852.review.test-alv.no

**Comment by klyngen** [comment]
💻 https://testimonia-852.review.test-alv.no

File: packages/website/src/templates/category.js (hunk 2 above)
Comment (mikkelklokkerud) [reply packages/website/src/templates/category.js:107]: I wanted the same approach but I wasn't allowed to use string interpolation in a static page query 🤔 

Screenshot 2023-02-17 at 15 48 38

File: packages/website/src/components/testimonialsSlider/index.js (hunk 1 above)
Comment (mikkelklokkerud) [reply packages/website/src/components/testimonialsSlider/index.js]: Updated for this

**Comment by klyngen** [comment]
💻 https://testimonia-852.review.test-alv.no

**Review by klyngen — APPROVED** [review]

**Comment by klyngen** [comment]
💻 https://testimonia-852.review.test-alv.no

**Pull request merged**